### PR TITLE
Metrics docs refactoring with finalized wording

### PR DESCRIPTION
**Pull request by amartinez116**
Building from  [#1337](https://github.com/autolab/Autolab/pull/1337) to update the images and wordings in the metrics feature page in documentation to match current updates.

## Description

Along with updating the images, I updated the wording that appears in the docs. Mainly getting rid of instances where we refer to it as the "risk metrics" to "student metrics".

Updated metrics image:
image

Updated watchlist image:
image

Updated student metrics image:
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

## Other issues / help required

If unsure, feel free to submit first and we'll help you along.

### CODE REVIEW

File: docs/features/metrics.md [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
 
 
 ## Usage Flow
 
 We envision the feature to be used by the instructors in this order. They would:
 
 1. Set up their course and assignments as per usual
-1. Set up [Risk Metrics](#risk-metrics) at the start of the course
-1. Be notified of new at risk students in their [Watchlist](#watchlist) on their course page
+1. Set up [Student Metrics](#Student-Metrics) at the start of the course
```
Comment (ashleyzhang) [inline docs/features/metrics.md:13]: Seems like the link (#Student-Metrics) needs to be all lowercase (#student-metrics) for it to work.

File: docs/features/metrics.md [hunk 2]
```diff
@@ -2,28 +2,28 @@
...
+1. Be notified of pending students in need of attention in their [Watchlist](#watchlist) on the course page
 1. Visit the [Watchlist](#watchlist), contact students if necessary, using it as a work list at the same time
-1. Refine the [Risk Metrics](#risk-metrics) as the course progresses
+1. Refine the [Student Metrics](#Student-Metrics) as the course progresses
```
Comment (ashleyzhang) [inline docs/features/metrics.md:16]: Same as above (#Student-Metrics should be all lowercase)

**Review by ashleyzhang — CHANGES_REQUESTED** [review]
Just a small change needed for the links to the student metrics section. Everything else looks great!

File: docs/features/metrics.md (hunk 1 above)
Comment (amartinez116) [reply docs/features/metrics.md]: Thanks for pointing that out! Should be fixed now

**Review by ashleyzhang — APPROVED** [review]
LGTM!

**Pull request merged**